Create Australian Archaeology (2023 Update).csl

CameronMcVicar opened the pull request:
Referencing Style based on the Australian Archaeological Association referencing system

adam3smith commented:
Thanks!
Could you say more about what you're trying to do? If the existing Australian Archaeology style is either incorrect and/or outdated, we'd want changes directly to that file/style. If this is something else, could you explain?

CameronMcVicar commented:
Hi,

The existing Australian Archaeology style available on the repository is incorrect in several ways. Mostly with the in-text references, for example, although it is author-date it requires a page number separated by a colon without a space and no page label (ie. p. Or pp.). So should appear as (Evans and Taylor 1999:56). I have updated these elements from the old style using the existing code. I then updated some of bibliography elements. Such as, no "editor" label after the authors in edited books; and, separating publishing-location and publisher by a colon not a comma. I think this is everything I've changed, the rest of the code remains the same.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Sebastian Karcher ***@***.***>
Sent: Sunday, June 11, 2023 11:57:59 AM
To: citation-style-language/styles ***@***.***>
Cc: CameronMcVicar ***@***.***>; Author ***@***.***>
Subject: Re: [citation-style-language/styles] Create Australian Archaeology (2023 Update).csl (PR #6570)


Thanks!
Could you say more about what you're trying to do? If the existing Australian Archaeology style is either incorrect and/or outdated, we'd want changes directly to that file/style. If this is something else, could you explain?

—
Reply to this email directly, view it on GitHub<https://github.com/citation-style-language/styles/pull/6570#issuecomment-1585978619>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BAOAYWANG6JF2FDKE2JFD7LXKUUK7ANCNFSM6AAAAAAZAGFAWM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

POBrien333 commented:
This looks like an old PR and the style has received updates in 2024 by another user in https://github.com/citation-style-language/styles/pull/7115
Closing this.
